app/views/static/terms.html.erb

```diff
@@ -0,0 +1,162 @@
+<h1>Terms of Use for <%= t('sufia.product_name') %></h1>
```

Comment:
Do we still need to add this file? Hard to see the diff, I'm guessing you've made some changes to the text?